CHANGELOG.md

```diff
@@ -15,6 +15,26 @@ FEATURES:
 * Kubernetes 1.24
   * Add support for Kubernetes 1.24 where ServiceAccounts no longer have long term JWT tokens. [[GH-1431](https://github.com/hashicorp/consul-k8s/pull/1431)]
   * Upgrade kubeVersion in helm chart to support Kubernetes 1.21+.
+  NOTE: Users deploying multiple services to the same Pod (multiport) must also deploy an Kube Secret for each ServiceAccount associated with the Consul service. The name of the Secret must match the ServiceAccount name and be of type 'kubernetes.io/service-account-token'
```

Comment:
* looks like there's two FEATURES sections?
* Can you put this under a BREAKING CHANGES section. Is this only needed if you're using Kube 1.24+? We should note that if true.

Reply:
Updated it in the breaking changes section.